webpages/css/main.css

```diff
@@ -183,6 +184,14 @@ main.index > img.scrollhint.hide {
   opacity: 0;
 }
 
+main.index > section.columns > section > h2.second_title {
```

Comment:
this block of code is from a different PR, isn't it? If it is, please remove it from here.